### ISSUE TITLE
SC 1.1.1 - removal of gendered language

### ISSUE THREAD

**Issue by GrahamTheDevRel** [w3c/wcag#3676]
## Overview / motivation
To improve the inclusivity of WCAG it would be good to remove all references to "he or she" and replace it with "they" (where it will not cause confusion of singular vs plural of course). 

## Current
Within the ["Intent" section of SC 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html#intent) there is the following sentence:
> A person who cannot hear an audio file can have the text alternative displayed so that **he or she** can read it.

## Proposed
I propose changing this to:
> A person who cannot hear an audio file can have the text alternative displayed so that **they** can read it.

## Note
As this is my first issue on 2.1 docs I am not sure what tags I should have added so can someone please adjust / correct, thanks.

**Comment by patrickhlauke** [comment]
There are also cases where just "he" on its own has been used - particularly in techniques ([F9](https://www.w3.org/WAI/WCAG22/Techniques/failures/F9), [F88](https://www.w3.org/WAI/WCAG22/Techniques/failures/F88), [G101](https://www.w3.org/WAI/WCAG22/Techniques/general/G101), [G112](https://www.w3.org/WAI/WCAG22/Techniques/general/G112), [G115](https://www.w3.org/WAI/WCAG22/Techniques/general/G115)) and the [example provided for extended audio descriptions](https://www.w3.org/WAI/WCAG22/Understanding/extended-audio-description-prerecorded.html#examples)

That should also be addressed/generalised.

**Comment by bruce-usab** [comment]
@GrahamTheDevRel it is okay if not, but are you comfortable making such edits via a pull request?

**Comment by patrickhlauke** [comment]
heck, i've got a spare minute, i'll do a PR (though in the past i've had a bit of pushback regarding singular "they")

**Comment by patrickhlauke** [comment]
done. some cases were actually ok, as they were direct quotes (e.g. from Pride and Prejudice)

https://github.com/w3c/wcag/pull/3680

**Comment by GrahamTheDevRel** [comment]
Patrick with the "quick draw" there! 🤣

I will go and do a review! 💪🏼💗

**Comment by mbgower** [comment]
Removing from project since we are now tracking the PR that resolves it.
